### PR TITLE
Graphql enums re-using the protos enums.

### LINKED ISSUE #5822: Rename graphql enums

### PR DESCRIPTION
**Pull request by jeremyletang**
close #5822

### CODE REVIEW

File: datanode/gateway/graphql/schema.graphql [hunk 1]
```diff
@@ -654,18 +654,18 @@ type Query {
...
+  STATUS_DONE
   "Indicates a transfer rejected by the Vega network"
-  Rejected
+  STATUS_REJECTED
   """
   Indicates a transfer stopped by the Vega network
   e.g: no funds left to cover the transfer
   """
-  Stopped
+  STATUS_STOPPEDah
```
Comment (dexturr) [inline datanode/gateway/graphql/schema.graphql]: chore (blocking): This doesn't seem right

File: datanode/gateway/graphql/schema.graphql [hunk 2]
```diff
@@ -724,14 +724,13 @@ type RecurringTransfer {
...
 
-  MarketTradingValue
+  DISPATCH_METRIC_MAKER_FEES_RECEIVED
 
-  MakerFeesReceived
+  DISPATCH_METRIC_LP_FEES_RECEIVED
 
-  TakerFeesPaid
-
-  LPFeesReceived
```
Comment (dexturr) [inline datanode/gateway/graphql/schema.graphql:734]: question (non-blocking): some of these have disappeared, i assume no longer needed?

**Review by dexturr — CHANGES_REQUESTED** [review]

File: datanode/gateway/graphql/schema.graphql (hunk 1 above)
Comment (jeremyletang) [reply datanode/gateway/graphql/schema.graphql]: yes seen and updated that already when reviewing.

File: datanode/gateway/graphql/schema.graphql (hunk 2 above)
Comment (jeremyletang) [reply datanode/gateway/graphql/schema.graphql:734]: Incorrect, they are still all here?

Comment (dexturr) [reply datanode/gateway/graphql/schema.graphql:734]: yep ignore me

**Review by dexturr — APPROVED** [review]

**Review by ValentinTrinque — APPROVED** [review]

**Pull request merged**